### PR TITLE
recoil and new gunmod

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Recoil maxium has been raised to 50 from 30 meaning it can build up more

Red is current:
Yellow is the inbettewn:
Green is now:
image

New gunmod - Pain-maker massively reduces a guns damage, AP and increase recoil offset and paindamage for when you dont want to snipe from 5000 years away and still take down someone

Fixes some bad numbers with kidneys
Fixes some bad trade names from organs
Makes the gun ui slightly bigger by default to prevent a UI scaling issue

Testing: shoot a gun a lot with the mod and without the mod to see recoil changes to make sure they were not storm trooper levels
Performance: no real change

### CODE REVIEW

**Comment by ApplePlastic** [comment]
Shooting with that much spread looks pretty goofy, I must say. It's alright. 

Mod is cool and something I haven't seen before yet, but does the pain damage stack when you use rubber bullets?

**Comment by Trilbyspaceclone** [comment]
It basically only works with rubbers, HPs and other pain-based bullets
as it scales with agony damage

**Review by hex37 — APPROVED** [review]

**Comment by hex37** [comment]
Approved!

File: code/modules/projectiles/guns/mods/mods.dm [hunk 1]
```diff
@@ -26,6 +26,33 @@
...
+	icon_state = "silencer"
+	price_tag = 100
+
+/obj/item/gun_upgrade/muzzle/pain_maker/New()
+	..()
+	var/datum/component/item_upgrade/I = AddComponent(/datum/component/item_upgrade)
+	I.weapon_upgrades = list(
+		UPGRADE_BULK = 2,
+		GUN_UPGRADE_STEPDELAY_MULT = 1.5,
+		GUN_UPGRADE_DAMAGE_MULT = 0.8,
```
Comment (TheSleepyCatmom) [inline code/modules/projectiles/guns/mods/mods.dm:44]:
```suggestion
		GUN_UPGRADE_DAMAGE_MULT = 0.5,
```
This should be a lot lower, as shown, to relaly put into effect that it should be used more with rubbers or sopo bullets and less with actual bullets. You can offset a 0.8 pretty easily by putting other mods on after it in different places.

File: code/modules/projectiles/guns/mods/mods.dm [hunk 2]
```diff
@@ -10,7 +10,7 @@
 //Silences the weapon, reduces damage multiplier slightly, Legacy port.
 /obj/item/gun_upgrade/muzzle/silencer
 	name = "Silencer"
-	desc = "A threaded silencer that can be attached to the muzzle of certain guns. Vastly reduces noise, but impedes muzzle velocity."
```
Comment (TheSleepyCatmom) [inline code/modules/projectiles/guns/mods/mods.dm:13]: Why was this removed? It does decrease damage a bit.
